nemo_skills/inference/check_contamination.py

```diff
@@ -29,19 +31,19 @@
 
 @nested_dataclass(kw_only=True)
 class CheckContaminationConfig(GenerateSolutionsConfig):
-    """Top-level parameters for the script"""
+    """LLM-based check contamination parameters. 
```

Comment:
Eventually would be good to add a gpu test for this script (and retrieve_similar). If it's too much for this PR, feel free to just create an issue so that we can track this

Reply:
Let me work on this in a separate PR. Created an issue here - https://github.com/NVIDIA/NeMo-Skills/issues/481